package.json

```diff
@@ -7,5 +7,9 @@
   "license": "MIT",
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "scripts": {
+    "server-build": "yarn workspace @online-games-shop/server build",
+    "server-watch": "yarn workspace @online-games-shop/server start"
```

Comment:
With proper workspace setup you can use something like this:

`"server-build": "yarn workspace server build",`